elements/rh-icon/demo/accessibility.html

```diff
@@ -0,0 +1,10 @@
+<rh-icon set="ui" icon="arrow-down" accessible-label="Page down"></rh-icon>
+
+<rh-icon icon="info" aria-labelledby="info" role="img"></rh-icon>
+<span id="info">Information</span>
```

Comment:
Nix aria-labelledby + role.

Would probably be good to have an example for people to copy/paste for {Icon} {Icon Label} (much like Marionne's "{Icon} Download" example in the docs). IMO that is an accessibility demo.